compiler/rustc_error_messages/locales/en-US/mir_build.ftl

```diff
@@ -192,9 +192,12 @@ mir_build_const_param_in_pattern = const parameters cannot be referenced in patt
 
 mir_build_non_const_path = runtime values cannot be referenced in patterns
 
-mir_build_unreachable_pattern = unreachable pattern
-    .label = unreachable pattern
-    .catchall_label = matches any value
+mir_build_unreachable_patterns = one or more patterns are unreachable
```

Comment:
```suggestion
mir_build_unreachable_patterns = {$amount ->
    [one] unreachable pattern
    *[other] multiple unreachable patterns
}
```

Looks a bit cleaner in my opinion. Take it or leave it though.

Reply:
Done